GOVERNANCE.md

```diff
@@ -0,0 +1,46 @@
+# Process for becoming a maintainer
+
+## Your organization is not yet a maintainer
+
+* Express interest to the senior maintainers that your organization is interested in becoming a maintainer. Becoming a maintainer generally means that you are going to be spending substantial time (>25%) on Amundsen for the foreseeable future. Ultimately your goal is to become a senior maintainer that will represent your organization.
+* We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing senior maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 2-3 months of working together and making sure we see eye to eye, the existing senior maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but 2-3 months is the approximate goal.
+
+## Your organization is currently a maintainer
+
+* First decide whether your organization really needs more people with maintainer access. Valid reasons are "blast radius", a large organization that is working on multiple unrelated projects, etc.
+* Contact a senior maintainer for your organization and express interest.
+* Start doing PRs and code reviews under the guidance of your senior maintainer.
+* After a period of 1-2 months the existing senior maintainers will discuss granting "standard" maintainer access. This will require 3 (senior) maintainers +1 vote and no -1 vote.
+* "Standard" maintainer access can be upgraded to "senior" maintainer access after another 1-2 months of work and another conference of the existing senior committers.
+
+## Maintainer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community.
+* Triage build issues - file issues for known flaky builds or bugs, and either fix or find someone to fix any master build breakages.
+* During GitHub issue triage, apply all applicable [labels](https://github.com/lyft/amundsen/labels) to each new issue. Labels are extremely useful for future issue follow up. Which labels to apply is somewhat subjective so just use your best judgment.
+* Make sure that ongoing PRs are moving forward at the right pace or closing them.
+* In general continue to be willing to spend at least 25% of ones time working on Amundsen (~1.25) business days per week).
+
+
+## When does a maintainer lose maintainer status
+
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of the maintainers per the voting process below.
```

Comment:
```suggestion
If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to emeritus status. In extreme cases, this can also occur by a vote of the maintainers per the voting process below.
```